src/middlewares/index.ts

```diff
@@ -0,0 +1,60 @@
+import { Middleware } from './types'
+import { ActionTypes as PendingOrdersActionTypes } from 'reducers-actions/pendingOrders'
+
+import { setStorageItem } from 'utils'
+import { GP_PENDING_ORDER_KEY } from 'const'
+import { GlobalState } from 'reducers-actions'
+
+const logger: Middleware = getState => next => (action): void => {
+  process.env.NODE_ENV === 'development' &&
+    console.debug(`
```

Comment:
use logDebug

Also, there's already a logging middleware, right? why do we want to do our own?

Reply:
true!

Reply:
artifact from testing anways, ill remove